### ISSUE TITLE
Separator icons not aligning properly

### ISSUE THREAD

**Issue by justinbarclay** [domtronn/spaceline-all-the-icons.el#47]
Hello,

I'm having an issue where the icons don't align to the mode-line properly. This happens regardless of the separator icons I use. If you have have any suggestions how I can resolve that issue it would be greatly appreciated.

This is on Emacs 26.0.50 on macOS Sierra (10.12.5)
![image](https://user-images.githubusercontent.com/6868388/27891454-428a5658-61b7-11e7-9be5-23c8e43250ae.png)

**Comment by domtronn** [comment]
Hi @justinbarclay, I'm _(unfortunately)_ very aware of this bug! It's an issue with the icon font I created and I've tried to fix it a bunch of times but whenever I do, it causes my laptop to crash 😅 

I'm not sure why, but I can't seem to make changes to my custom font without it breaking... I'll pick it up again when I have a free evening and fingers crossed I can fix it this time! :v:

**Comment by cmacrae** [comment]
@domtronn Not meaning to pester, just a polite notice that I'm also seeing this and reported it in a separate issue: #59 (duplicate which is now closed, but the info may be helpful 🙂 )

**Comment by domtronn** [comment]
Hi @cmacrae, I know what the fix for this is... I basically borked the creation of the wavy separators when I was making my font, so I just need to update the glyphs and rebuild the new font.

... Which I've tried a few times 😅 I use [IcoMoon](https://icomoon.io/app/) to build the fonts, and it seems to work, but when I update with new/better separators, Emacs segment faults when trying to parse the new hex codes for the characters. I've tried a few other font building apps, but ran into different issues

Basically, whenever I try to make the new font, it breaks and I don't know why! It's annoyingly quite a time consuming process as well, so I've struggled to revisit it.

When I find a spare weekend I'll be able to properly commit to fixing this 👍

**Comment by cmacrae** [comment]
@domtronn Sounds annoyingly fiddly! Cheers for the update. Hopefully it's not too much of a headache 😅
